modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+// `blockstorage` conditionals are used (declared in the "*-block" assembly) to separate content
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
+
+.Prerequisites
+
+For smart-cloning to occur:
+
+* Your storage provider must support snapshots.
+* The source and target PVCs must be defined to the same namespace.
+* The source and target PVCs must be defined to the same storage class.
+* The VolumeSnapshotClass object must reference the storage class defined to both the source and target PVCs.
+
+If any of these prerequisites are not met, host-assisted cloning occurs when you create a DataVolume with a
+PVC source.
+
+.Procedure
+
+To initiate cloning of a DataVolume:
+
+. Create a YAML file for a DataVolume object that specifies the name of the
+new DataVolume, the name and namespace of the source PVC,
+ifdef::blockstorage[]
+`volumeMode: Block` so that an available block PV is used,
+endif::[]
+and the size of the new DataVolume.
++
+For example:
++
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
```

Comment:
Should use `v1beta1` for DataVolumes now

Reply:
Changed. Thanks!